library/nametab.ml

```diff
@@ -378,6 +368,30 @@ module UnivIdMap = HMap.Make(UnivIdOrdered)
 type univrevtab = full_path UnivIdMap.t
 let the_univrevtab = Summary.ref ~name:"univrevtab" (UnivIdMap.empty : univrevtab)
 
+(* Module-related nametab *)
+
+type modules_nametab = {
+  modtypetab : MPTab.t;
+  modtab : MPDTab.t;
+  dirtab : DirTab.t;
+  modrevtab : mprevtab;
+  modtyperevtab : mptrevtab;
+}
```

Comment:
Similar comment on maybe using `Modules.t` ?

Reply:
Sure, how would you call the `modules_nametab` ref, in this style?

Reply:
It will be `Modules.nametab`.

Reply:
Done.